website/src/components/Layout.astro

```diff
@@ -18,7 +18,7 @@ const { sections } = Astro.props;
 <html lang='en'>
   <head>
     <meta charset='utf-8' />
-    <link rel='icon' href=`${import.meta.env.BASE_URL}/favicon.ico` type='image/x-icon' />
+    <link rel='icon' href=`/favicon.ico` type='image/x-icon' />
```

Comment:
Do you know why `import.meta.env.BASE_URL` doesn't work??

Reply:
It's because it added an extra `/`, so it broke the link.

Reply:
Does this work then?
```suggestion
    <link rel='icon' href=`${import.meta.env.BASE_URL}favicon.ico` type='image/x-icon' />
```

Reply:
If it doesn't work, I can just merge the PR as is, assuming there's a bug or a limitation in Astro.
https://docs.astro.build/en/reference/configuration-reference/#base

Reply:
I expect it would work, but not what astro suggests, since the domain is not github anymore and it's a proper domain now. So what we currently have would work, and it's clean. Let's merge and see if things work, i'll fix again for sure.

Reply:
I'm very curious why `BASE_URL` doesn't work, but it doesn't need to block merging this PR. 👍